internal/test/cluster.go

```diff
@@ -88,6 +89,34 @@ func Bundles(t *testing.T) *releasev1alpha1.Bundles {
 	return bundles
 }
 
+func EksdRelease(t *testing.T) *eksdv1alpha1.Release {
```

Comment:
nit: you can add this to improve logs

```suggestion
func EksdRelease(t *testing.T) *eksdv1alpha1.Release {
  t.Helper()
```